src/BuildIt.Forms/BuildIt.Forms.Controls/CameraPreviewControl.xaml.cs

```diff
@@ -36,6 +38,8 @@ public CameraPreviewControl()
         /// <returns>The path to the saved photo file</returns>
         internal delegate Task<string> CaptureNativeFrameToFile(bool saveToPhotosLibrary);
```

Comment:
Can we tidy up CaptureNativeFrameToFileDelegate to be consistent with RetrieveSupportedFocusModesFunc ie remove CaptureNativeFrameToFile and change CaptureNativeFrameToFileDelegate to be Func<Task<string>> instead.
Also, not sure I like the naming convention of using "Func" on the end - is this a naming convention we came up with, or is it widely used within XF?

Reply:
Yep can clean it up. I put the Func suffix because without it, it'll conflict with the actual method that is public